### PR TITLE
v1.4 Update Documentation

### LINKED ISSUE #162: v1.4 Update Developer Guide
Currently, the developer guide is not showing the activity diagram for EditEventCommand.

### PR DESCRIPTION
**Pull request by guohuang88**
Update the User Guide issue raise in previous PR (https://github.com/AY2324S2-CS2103-F09-2/tp/pull/238).
Update screenshot for DeleteCommand and ClearCommand
Add EditEventActivityDiagram in Developer Guide.
Fixes #162

### CODE REVIEW

**Review by ZhiWei1010 — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -625,13 +625,17 @@ The `EditEventCommand` class is responsible for editing a specific Event for a p
 
 #### Specifications
 
-* EditEventCommand takes in four parameters: `PATIENT_INDEX`, `EVENT_INDEX`, `EVENT_NAME` and `DATE`.
-  All parameters are compulsory. Optionally, you may include `TIME` in the `DATE` parameter.
+* EditEventCommand takes in four parameters: `PATIENT_INDEX`, `EVENT_INDEX`, `NAME_OF_EVENT`
```
Comment (SherisseTJW) [inline docs/DeveloperGuide.md:628]: Perhaps give example of with and without the `TIME`?

**Review by SherisseTJW — COMMENTED** [review]
Just a minor suggestion for the DG in the Edit Event section, otherwise, generally LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (guohuang88) [reply docs/DeveloperGuide.md:628]: okay! Will do it in my next PR since this PR is merged! Thanks for the suggestion:)

**Pull request merged**